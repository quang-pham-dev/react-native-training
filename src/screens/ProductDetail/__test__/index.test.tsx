import React from 'react'

// LIBS
import {render} from '@testing-library/react-native'

// Screens
import {ProductDetail} from '@screens'

const props: any = {
  navigation: {
    navigate: jest.fn(),
    setOptions: jest.fn(),
  },
}

describe('Product detail Screen', () => {
  test('should render correctly', () => {
    const tree = render(<ProductDetail {...props} />)
    expect(tree).toMatchSnapshot()
  })
})
