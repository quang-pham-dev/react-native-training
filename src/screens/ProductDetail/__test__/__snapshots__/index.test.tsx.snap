// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product detail Screen should render correctly 1`] = `
<RCTScrollView
  contentContainerStyle={
    Object {
      "flexGrow": 1,
    }
  }
  showsVerticalScrollIndicator={false}
  style={
    Array [
      Object {},
    ]
  }
>
  <View>
    <View
      style={
        Array [
          Object {
            "width": "100%",
          },
        ]
      }
    >
      <ActivityIndicator
        color="#3D93F8"
      />
    </View>
  </View>
</RCTScrollView>
`;
